_scripts/update.py

```diff
@@ -145,7 +150,7 @@ def update_str(
     with open(data_path, "w") as file:
         json.dump(messages, file, indent=2)
```

Comment:
If we're sorting the config, we should sort here too.

```suggestion
        json.dump(messages, file, indent=2, sort_keys=True)
```